src/components/Person/Person.jsx

```diff
@@ -1 +1,17 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+const partherInfo = details => {
```

Comment:
There is a typo in the function name `partherInfo`. It should be `partnerInfo` to accurately reflect its purpose.